nuxeo-web-ui-ftest/scripts/specs-processor.js

```diff
@@ -1,9 +1,18 @@
+ const Finder = require('fs-finder');
+ const fs = require('fs');
+
 module.exports = (argv) => {
+  let features = [];
   const args = require('minimist')(argv.slice(2));
-  if (args['specs']) {
-    return args['specs'].split(',');
+  if (args['cucumberOpts']['tags']) {
+    const files = Finder.from('./test/features').findFiles('*.feature');
+    files.forEach((file) => {
```

Comment:
couldn't you replace the whole `forEach` with something like 
`features = files.filter(file => fs.readFileSync(file, 'utf8').includes(args.cucumberOpts.tags));`? (wrote this from the top of my head, not sure if it works but looks cleaner to me.

Reply:
Yes, that works perfectly!